scripts/gen_help_html.lua

```diff
@@ -35,9 +35,7 @@ local invalid_links = {} ---@type table<string, any>
 local invalid_urls = {} ---@type table<string, any>
 local invalid_spelling = {} ---@type table<string, table<string, string>>
 local spell_dict = {
-  Neovim = 'Nvim',
   NeoVim = 'Nvim',
-  neovim = 'Nvim',
```

Comment:
What is the case for allowing `neovim`? When is it ever correct to use `neovim` (as opposed to "Neovim", which at least has correct capitalization)?

Reply:
I don't see why both can't be correct